### ISSUE TITLE
[Feature Request]: Only One Video Per Channel in Subscription Feed

### ISSUE THREAD

**Issue by bacontime** [FreeTubeApp/FreeTube#3127]
### Guidelines

- [X] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the feature request I want to file, without success.
- [X] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the feature request I want to file, without success.
- [X] This issue contains only one feature request.

### Problem Description

Some channels upload near-daily.
Other channels upload only once every few months.

In a feed which lists all the uploads from my subscriptions in reverse-chronological order, the latter are drowned out by the former.

### Proposed Solution

I would like a toggle or settings option which, when enabled, makes it so that the subscription feed displays _only the most recent video_ from each subscribed channel.

### Alternatives Considered

I sometimes avoid subscribing to a channel whose content I enjoy simply because they upload _too_ frequently.

Another option for a FreeTube user is to create a separate profile containing only subscriptions with infrequent uploads.

### Issue Labels

new optional setting

### Additional Information

This feature request was inspired by a similar UX paradigm on the web-fiction site Royal Road. 
Screenshot of Royal Road "Follow List":

![Screen Shot 2023-01-29 at 12 20 56 AM](https://user-images.githubusercontent.com/42920487/215309066-7a4e74af-9f66-4c72-a0b0-087b1c1e6fd0.png)

As can be seen in the above screenshot, Royal Road's subscription feed sorts stories by order of most recent update, but displays only a single block per story, with a link to the most recent chapter. This is very similar to how I would like my Youtube subscription feed to work.

How might this feature interact with the "Hide Videos on Watch" setting? I can imagine two options:
- Display only the most recent _unwatched_ video from each channel in the feed.
- or Simply do not display any videos from a channel if the most recent one has been watched.

I have no strong opinion on which implementation would be better.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
duplicate of https://github.com/FreeTubeApp/FreeTube/issues/1428
